nebula.config.js

```diff
@@ -2,6 +2,7 @@
 module.exports = {
   build: {},
   serve: {
+    open: false,
```

Comment:
why do you want to open the app?

Reply:
I don't want to open. Personally I think it is annoying when it opens a new window every time you start the nebula server. Often I still have a page open actually. For running the integration tests locally it also does not make sense and currently they share the config